src/data/camera.ts

```diff
@@ -38,22 +38,25 @@ export const computeMJPEGStreamUrl = (entity: CameraEntity) =>
 
 export const fetchThumbnailUrlWithCache = (
   hass: HomeAssistant,
-  entityId: string
+  entityId: string,
+  ...args: any[]
 ) =>
   timeCachePromiseFunc(
     "_cameraTmbUrl",
     9000,
     fetchThumbnailUrl,
     hass,
-    entityId
+    entityId,
+    args
   );
 
 export const fetchThumbnailUrl = async (
   hass: HomeAssistant,
-  entityId: string
+  entityId: string,
+  ...args: any[]
 ) => {
   const path = await getSignedPath(hass, `/api/camera_proxy/${entityId}`);
-  return hass.hassUrl(path.path);
+  return hass.hassUrl(`${path.path}&width=${args[0][0]}&height=${args[0][1]}`);
```

Comment:
The reason you had double index the array (`[0][0]`) is because `fetchThumbnailUrlWithCache` didn't spread the args.

```suggestion
  return hass.hassUrl(`${path.path}&width=${width}&height=${height}`);
```

Reply:
fyi I have not tested my suggestions.